tests/runner/Cargo.toml

```diff
@@ -14,14 +14,19 @@ async-trait = "0.1.21"
 syn = { version = "1.0", features = ["full"]}
 rand = "0.8"
 md-5 = "0.9"
-serde = "1.0"
+serde = {version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+csv = "1.1"
 inventory = "0.1"
 tokio = { version = "1.4", features = ["macros"] }
 bencher = "0.1"
 prettytable-rs = "0.8"
 hdrhistogram = "7.3.0"
 async-lock = "2.4.0"
+async-channel = "1.6.1"
+sysinfo = "0.19.1"
```

Comment:
we should create different feature flag. By default, performance run be disabled.  Pulsar and Kafka run should be also made as optional. 

Another thought  is to create separate runner for performance